deploy/docs/Best_Practices.md

```diff
@@ -1095,3 +1096,39 @@ The reliability of the system can be calculated using the following formulas:
 
 - If limited by queue_size: `number_of_instances*timeout[min]*sending_queue.queue_size/load_in_DPM` minutes.
 - If limited by PVC size: `number_of_instances*PVC_size/(1KB*load_in_DPM)` minutes.
+
+## Assigning Pod to particular Node
+
+Kubernetes offers a feature of assigning specific pod to node. Using this feature we can bind them to linux nodes.
+In order to do that `nodeSelector` has to be used. By default node selectors can be set for below pods:
+
+```
+  fluent-bit.nodeSelector.kubernetes.io/os=linux
+  fluentd.events.statefulset.nodeSelector.kubernetes.io/os=linux
+  fluentd.logs.statefulset.nodeSelector.kubernetes.io/os=linux
+  sumologic.setup.job.nodeSelector.kubernetes.io/os=linux
+  kube-prometheus-stack.prometheus-node-exporter.nodeSelector.kubernetes.io/os=linux
+  kube-state-metrics.nodeSelector.kubernetes.io/os=linux
+  prometheusOperator.nodeSelector.kubernetes.io/os=linux
+  prometheus.prometheusSpec.nodeSelector.kubernetes.io/os=linux
+```
```

Comment:
Perhaps change this into a markdown table with name of the (sub)component like e.g. fluent-bit in the left column and the key for changing in `values.yaml` in the right?

Reply:
I agree with @pmalek-sumo, eventually as a yaml style configuration with comments